### PR TITLE
[MODULAR] DS-2 Armoury Overhaul & DS-2 Access Changes

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

weh3

See the title. Supplies with the DS-2 armoury with some le epic gamer loot, for the purposes of self-defence. Because AssOps isn't a thing anymore and will never come back.

List of gamer loot added:
- Some rifles/primaries
- Bulldog shotties
- Emags, doorjacks, and suppressors
- some funnie shields

I have also supplied the MAA with a Unica/Mateba revolver. The MAA is the ghetto DS-2 HoS, and on station, the HoS does get a lethal weapon, but the MAA doesn't? Confusing. The MAA also gets their own winter coat, much like the admiral has, which you can see above. Here's a sheet screenshot.

image

Also changes up the DS-2 firing range, to be more aesthetic.
image

Admeme and Memetainer approval:
junglerat
funce

## Access Changes

Three new tiers of Syndicate access, Syndicate Security, Syndicate Armoury, and Syndicate leader. Brig Officer's can no longer just raid the armoury, neither can the Deck Officer. Access is restricted to the MAA, and the Admiral. Syndicate Leader is now reserved for the DO's and Admiral's office.

## Proper Syndicate Headsets:

image

For DS-2 Security Operatives. The MAA, and Admiral get a command version with loudmode.

## Proper Syndicate IDs:

image

DS-2 now uses proper Syndicate IDs, instead of generic black ones. The Admiral has had his gold ID removed because it was cringe, and unnecessary.

## Captain Vest Recolouring:

image

Slight recolouring of the Syndicate Captain's vest, just so it looks better.

## KT45-C

image

Probably the biggest change, in an attempt to the big buff that is the DS-2 Armoury, Interdyne Operatives have been given a new bullpup. It shoots .32 and is nowhere as good as the Sniper they had was, but it's a gun, and it looks intimitading. Deck Officer still spawns with a regular Sniper.

```
Stats:
Calibre: .32
Mag cap: 32 rounds
Selective fire: SEMI, 3 ROUND BURST
```

## Some other changes:

- DS-2 Operatives spawn with duffle bags now. Two reasons for this, one to balance with the Marines coming in like, a while, which will have no slowdown duffles, and are the main force fighting DS-2. And Two, most DS-2 players already get no speed duffles by taking the surgery ones that spawn, this just makes it so they don't have to do that.
- MAA and Admiral's guns are now put in gunsets that spawn in their locker, it has spare ammo and an ammo box. This was already in the armoury, but I removed it to make more space.
- Beret's are now armour coded to be representative of the role's station counterpart, Admiral's beret now uses Captain's hat armour, and MAA's beret now uses HoS' beret armour.
- Also changed some spelling on Winterized Hardsuits (git gud nick)

## How This Contributes To The Skyrat Roleplay Experience

DS-2 is literally the Syndicate vessel for that sector of space, and they're given pistols and Sec shotguns? It doesn't make sense. So, I have fixed that.

This stuff is NOT for gaming the station and powergaming. DS-2/Interdyne cannot attack the station and chances are they never will be able to. It's an easy bwoink, and against the rules. These weapons are purely for self-defence, and for gawking at their beauty. Assuming people can't behave it, I will neuter slightly, but I'd rather not. As punishing all for the few's mistakes is stupid logic, imo.

## Changelog

:cl: axietheaxolotl
expansion: Complete overhaul of the DS-2 armoury, more powerful then before, but more restricted.
expansion: Three new tiers of Syndicate access, Syndicate Security, Syndicate Armoury, and Syndicate leader.
expansion: Interdyne researchers are now issued unique bullpup SMGs, significantly less powerful then sniper rifles, but with 300 bullets in total, surely you can kill something?
expansion: DS-2 Security now spawn with proper Syndicate bowmans, and the MAA and Admiral have been given special command ones.
expansion: DS-2 Admiral, and MAA now spawn with gunsets for their assigned weapons. With spare ammo and a ammo bag.
imageadd: Recolouring of the Syndicate Captain's vest, and a new winter vest for the MAA.
balance: DS-2 operatives now spawn with generic syndicate duffle bags, instead of backpacks.
tweak: DS-2 IDs now use proper Syndicate IDs, instead of generic black ones.
code: Removed old, unused DS-2 access trims
spellcheck: fixed non-proper nouns in Winterized BRHS hardsuit
/:cl:

### CODE REVIEW

**Comment by softcerv** [comment]
I'm not opposed to DS-2 and Interdyne getting more stuff, but please don't encourage more people to raid them. Most people playing ghost roles usually aren't picking them because they want to play defend the base from some space powergamer or a miner.

**Comment by SpaceVampire** [comment]
>DS-2. is also raidable, and you don't have to be an antag to do that. So, this will give both more incentive to raid it, and also give DS-2 more options to defend themselves, if need be.

Have you ever seen that happen? It isn't raid-able, staff would scream at anyone who tries it because you aren't allowed to touch ghost roles. I really don't think they need even more gamer loot, as it seems like the kind of stuff that can only lead to trouble. When the best outcome is that no one ever touches it.

**Comment by axietheaxolotl** [comment]
fuck

**Comment by ghost** [comment]
Hi, I am one of the people who actually main DS-2. 

One of the biggest things I tell people ICly off over is carrying their guns around outside in lavaland. The gamer gear looks great, and it'll give a decent amount of good defense to the average space explorer who decided to loot DS-2.

Policy will have to be implemented and current policy enforced. I.E, don't touch the gamer gear unless you actually have a good reason to touch it. People playing a ghost role should always read the policy even if it's extremely outdated (though it's being worked on)

I'm all down for people breaking into DS-2 and sneaking around. **But there needs to be people actively playing the roles which a quick ahelp should be sent out before you start breaking into the airlocks. There needs to be a statement which these players are also held at a higher standard**

I.E, I shouldn't run into the same assistant. Some of the best conflict to Roleplay is that of consiquences. If you get captured, play a few Interdyne prisoner rounds, and have it overall build into the arc of your character. 

Maybe it can become an option for the Vanguard to organize. They have some pretty gamer gear too.

This ghost role DOES get population. AssOps will never come back and I will always actively argue that you will never see them on or around the station or using any of this gear to wordlessly shoot miners. They shouldn't be carrying around these guns outside in the first place. 

With the current atmosphere of the Interdyne players, I genuinely feel that most of them are respectful in terms of upholding roleplay. We do infact roleplay with people and commonly keep staff in the light whenever conflict does occur.

**Comment by Stonetear** [comment]
I really want this to be added.  Interdyne is good because it gives people an appropriate place to test weapons and syndicate gadgets, as well as exchange advanced tips with other agents.

However, I should say the emag is exceptionally powerful in the right hands.  I absolutely want the armory to be stocked with emags and doorjacks, but maybe there should be a piece of paper warning people about getting into trouble?

**Comment by ghost** [comment]
> I really want this to be added. Interdyne is good because it gives people an appropriate place to test weapons and syndicate gadgets, as well as exchange advanced tips with other agents.
> 
> However, I should say the emag is exceptionally powerful in the right hands. I absolutely want the armory to be stocked with emags and doorjacks, but maybe there should be a piece of paper warning people about getting into trouble?

There's already policy set in place that you shouldn't touch the armory.

**Comment by GoldenAlpharex** [comment]
I'm not too sure about this, if I'm honest. Adding gamer loot for the sake of adding gamer loot is... Questionable, I'd say.

I would like a list of what they currently get in their armory at the moment, compared to this. Because what I see here feels excessive for stuff that's only added not to be touched.

**Comment by axietheaxolotl** [comment]
In an attempt of balancing, I've made the following changes to Interdyne:

- They don't all spawn with .50 cals. Only the Deck Officer does. One surplus spawns in the vault.
- Shaft Miners have been given makarovs to replace their .50s.
- Interdyne Kitchen now has a knife.

**Comment by axietheaxolotl** [comment]
> I would like a list of what they currently get in their armory at the moment, compared to this. Because what I see here feels excessive for stuff that's only added not to be touched.

image
This is what DS-2's armoury currently looks like.

**Comment by Ruediger4** [comment]
Howdy, i'm the dipshit that plays Ignis/the funny Syndicate lizard 99% of the time

> I would like a list of what they currently get in their armory at the moment, compared to this. Because what I see here feels excessive for stuff that's only added not to be touched.

**Current DS2 armory has**
-3 eswords
-3 telescopic batongs
-3 sets of riot armor + shields
-3 sets of standard armor (Brig Officers spawn with bulletproof vests)
-3 Riot shotguns
-3 Makarovs + and extra 3 clips and two 9mm ammo boxes
-2 Blood-red hardsuits
-1 ion gun + 1 temp rifle
-3 Syndicate jaws of life
-3 chest rigs and 3 operative holsters
-2 Bulldog mags (Admiral has a Bulldog shotgun in their glass thingy)
-2 Combat Glove Plus
-2 boxes of Syndicate Firing Pins
-4 Barrier grenades
-1 box of flashbang/teargas/handcuffs each
-Tracking implant box

So _this_ armory would add a new
- 3-6 Energy Shields? I think that's that on the corner left of the esword/batongs.
- One more spacesuit/hardsuit
- Reflector vest thingy
- Apparently 3 more sets of both standard and riot armor
- 3 Bulldog Shotguns
- Rifles and their respective ammoboxes
- 3 Rechargers inside Armory
- 2 more Combat Gloves Plus(?)
- Extra ammo clips for the Admiral's Stechkin APS + MaA's Revolver
- 1 duffelbag with Emags
- 1 duffelbag with doorjacks
- 1 duffelbag with surplus/cheap supressors
- Not sure what comes in the 4th duffelbag, me suck at reading changedfiles. Probably a few more thingies

While apparently removing

- Both riot shotguns and Makarovs
- The Makarov clips + the 9mm ammoboxes
- The riot shields

Master at Arms and Brig Officers spawn **only with a stun baton** and (i believe only the MAA) with a box of spare handcuffs in their bag. Each of their lockers have some basic sec tools (Secbelt and like, a few more things I believe? But no weapons/disablers/anything like that).

Keep in mind that besides Interdyne crew and the Admiral in DS2, **no one in DS2 spawns with any sort of guns.** Not BO's, not MaA, not Staff or Operatives which are just reskinned assistants. If there's no Admiral, Deck Officer, Brig Officer or Master at Arms, no one can access the Armory unless they decide to hack in. 

**_(Past this is my 2 cents regarding current state of DS2 with the current armory + my fears with this new armory)_**

The lack of gamer loot really isn't an issue since I can count the number of times someone broke into DS2 beyond "By accident" or "Me have no oxygen please help" with the fingers in one hand, and the only situation where we actually _fired_ at someone was to get the point across that we weren't gonna talk to them if they brought a cyborg to bolt open doors inside DS2. If ya tell'em off, they **usually** understand for fear of pissing off admemes, even though we're fine and have fun telling people to GTFO because roleplay is fun.

_**THAT BEING SAID;**_ as much as I love the gamer loot, we've had issues with people lately either: 
**A) Not reading the basic _flavortext_ of any of those roles** and assuming we were basically Nuke Ops lite, wanting to fuck over NT at **any** given chances or just doing stuff without knowing that you're supposed to maintain a degree of secrecy regarding the true nature of Interdyne
 **B) People joining as roles with Armory access just to treat it as a candy store and grab a shitload of stuff** only to then either fuck off to space, fuck off to Lavaland, or fuck off to ERP someone in the dorms.
 **Bonus points if they join is as Brig Officer or Master at Arms, grab the gamer loot, then _complain when you tell them to do their job and babysit the prisoners._**

I have mixed feelings about this because as much as an actual bigger armory sounds like fun and can probably mean funnier interaction with admemes if they run more aggressive events in DS2/Interdyne + also more fitting to the Syndicate vibes, it'll also mean people just joining as their Station character/Random human to yoink gamer loot and then just fuck off with it.

The situation I mentioned is _**that**_ specific because it has happened. At least once.
Pretty sure more times when I wasn't on, as well.

Genuinely **the least of the worries here should be space raiders and more about the people who see gamer loot as a ghost and join only to grab it and then cause issues.** After all, 99% of the time at least **someone** is awake and they can just ahelp if someone broke in while DS2 staff was asleep or in VERY low numbers.

**TL;DR If the Ghost Role Policy becomes more enforced and the DS2 roles taken a _tiny_ bit more seriously, I'm all in for it. Otherwise, I'd say it's fine in it's current state.**

**Comment by RatFromTheJungle** [comment]
I don't know about removing the snipers from the planet side base, as ideally they shouldn't be looting the up-top armory, and they exist as the 'do not invade' warning to miners, without it, miner's will generally be at an advantage against interdyne. (Premium pen's can functionally outheal the makarovs.)

**Comment by ghost** [comment]
> I don't know about removing the snipers from the planet side base, as ideally they shouldn't be looting the up-top armory, and they exist as the 'do not invade' warning to miners, without it, miner's will generally be at an advantage against interdyne. (Premium pen's can functionally outheal the makarovs.)

There would be two snipers left on Interdyne. Plus pens are not instant in atmosphere.

**Comment by axietheaxolotl** [comment]
I've got more stuff to do

**Comment by axietheaxolotl** [comment]
I got some stuff to do 🦖

**Comment by GoldenAlpharex** [comment]
I don't know, honestly, after reading this and thinkig about it, I don't really think this is a great change overall. Gamer loot bloat just because is not something I'm very fond of.

I'll let @jjpark-kb give his opinion, but if there's no support from him, I'll close this tomorrow or Thursday.

**Comment by Ruediger4** [comment]
> I don't really think this is a great change overall. Gamer loot bloat just because is not something I'm very fond of.

Honestly some of the gamer loot is a bit too much, but other stuff could genuinely be a welcome addition due to the fact that current armory barely has enough things to arm up to 9 other staff with lethals or melee weapons capable of lethally neutralizing threats (3 shotguns, 3 makarovs and 3 eswords. Only DS2 staff that spawns with weapons is Admiral, which is an APS and the Bulldog in their office).

Granted, the cases where people need to get armed are rare, but other times it's needed (funny xenomorph infestation that one time).

Maybe less bloat antag stuff and more just... guns, maybe? I don't see doorjacks or emags being that necessary at all tbh, and x4/c4 can be a bit overkill, but even just 1 or 2 more makarovs can be enough.

**Comment by GoldenAlpharex** [comment]
> > I don't really think this is a great change overall. Gamer loot bloat just because is not something I'm very fond of.
> 
> Honestly some of the gamer loot is a bit too much, but other stuff could genuinely be a welcome addition due to the fact that current armory barely has enough things to arm up to 9 other staff with lethals or melee weapons capable of lethally neutralizing threats (3 shotguns, 3 makarovs and 3 eswords. Only DS2 staff that spawns with weapons is Admiral, which is an APS and the Bulldog in their office).
> 
> Granted, the cases where people need to get armed are rare, but other times it's needed (funny xenomorph infestation that one time).
> 
> Maybe less bloat antag stuff and more just... guns, maybe? I don't see doorjacks or emags being that necessary at all tbh, and x4/c4 can be a bit overkill, but even just 1 or 2 more makarovs can be enough.

Exactly, I'm not mad about having weapons for defense, but those weren't the main point of my "gamer loot" remark, you hit the nail right on the head. Doorjacks and emags serve no purpose there, you don't need suppressors, you don't need more combat gloves+, you don't really need a reflector vest (It's actually a theft objective, no matter what happens, this one just isn't staying), etc.

**Comment by GoldenAlpharex** [comment]
Oh, and, I feel like I'd rather have them wear Blood-red hardsuits rather than riot gear, that's a loooot more thematic.

**Comment by Ruediger4** [comment]
> you hit the nail right on the head.

_Bless, I finally am useful for something._

Jokes asides do you think it'd genuinely be okay if the armory was less **"Antag Candy Store"** and more _**"Actual syndicate armory with guns instead of infiltration equipment"**_?

Would removing anything that's for hacking, break and entering or doing the sneaky **(Emags, doorjacks, supressors, reflect vest, maybe also the x4/c4?)** in favor of more actual "Remove intruder" **(guns, leaving rechargers, substituting some armor for hardsuits??)** be more balanced if so?
_Also the issue with hardsuits is having to make more suit storage units, which will take some space and rearrangement of the armory if we're actually adding more._

> Oh, and, I feel like I'd rather have them wear Blood-red hardsuits rather than riot gear, that's a loooot more thematic.

My main pet peeve w/that is that Riot armor is genuinely useful due to the fact that shoving is currently broken rn.

On the other hand; it do be kinda broken when from 10 instances of DS2 fighting something, 7 out of 10 will be melee, whether it is prisoners getting uppity or xenos or anything that has to get close to harm you.

**Comment by Ruediger4** [comment]
Also there's the tangent in the air about the currently-in-development-again Ghost Role Policy which, if it stays as it is currently, will allow DS2 to do some minor sabotaging operations against the station. If so, some of the equipment would have to be reviewed or "infiltration" supplies to be added at a very _very_ limited amount instead of duffelbag-worth quantities and only allowed to be used for those situations.

Then again that's sabotaging that requires ghost roles to ahelp about it and admins to explicitly allow it and set the limitations properly so I don't see it as an issue but something that could happen at some time in the future.

**Comment by axietheaxolotl** [comment]
kay, so added some stuff:

## Access Changes

Three new tiers of Syndicate access, Syndicate Security, Syndicate Armoury, and Syndicate leader. Brig Officer's can no longer just raid the armoury, neither can the Deck Officer. Access is restricted to the MAA, and the Admiral. Syndicate Leader is now reserved for the DO's and Admiral's office.

## Proper Syndicate Headsets:

image

For DS-2 Security Operatives. The MAA, and Admiral get a command version with loudmode.

## Proper Syndicate IDs:

image

DS-2 now uses proper Syndicate IDs, instead of generic black ones. The Admiral has had his gold ID removed because it was cringe, and unnecessary.

## Captain Vest Recolouring:

image

Slight recolouring of the Syndicate Captain's vest, just so it looks better.

## KT45-C

image

Probably the biggest change, in an attempt to the big buff that is the DS-2 Armoury, Interdyne Operatives have been given a new bullpup. It shoots .32 and is nowhere as good as the Sniper they had was, but it's a gun, and it looks intimitading. Deck Officer still spawns with a regular Sniper.

```
Stats:
Calibre: .32
Mag cap: 32 rounds
Selective fire: SEMI, 3 ROUND BURST
```

## Some other changes:

- DS-2 Operatives spawn with duffle bags now. Two reasons for this, one to balance with the Marines coming in like, a while, which will have no slowdown duffles, and are the main force fighting DS-2. And Two, most DS-2 players already get no speed duffles by taking the surgery ones that spawn, this just makes it so they don't have to do that.
- MAA and Admiral's guns are now put in gunsets that spawn in their locker, it has spare ammo and an ammo box. This was already in the armoury, but I removed it to make more space.
- Beret's are now armour coded to be representative of the role's station counterpart, Admiral's beret now uses Captain's hat armour, and MAA's beret now uses HoS' beret armour.
- Also changed some spelling on Winterized Hardsuits (git gud nick)

**Comment by axietheaxolotl** [comment]
I've still got some loot to change in the armoury, hence why this is still a draft. But, yeah.

**Comment by axietheaxolotl** [comment]
oh that's how

**Comment by axietheaxolotl** [comment]
it did not fix CI

**Comment by axietheaxolotl** [comment]
oh **_joy_**

**Comment by axietheaxolotl** [comment]
Hey, so. After talking with friends, I've changed the armoury up a bit. It's no longer Traitor Candystore, it's now unga gun man. It's heavily focused on guns, and being an actual armoury. The main reason for this is @dawsonkeyes' marines, which will be the main antagonising force when they eventually come out. Which have big gamer loot.

image

This, is what the armoury looks like now. With the addition of more guns, and two new Cybersun hardsuits. I've also removed the trenchcoat, the emags, doorjacks, and suppressors. Hopefully this is satisfactory.

**Comment by axietheaxolotl** [comment]
what
how did DS-2 die

**Comment by GoldenAlpharex** [comment]
I feel like this is a lot of changes all bundled up in the same PR, most of which should've been split into different. By that, I mean: Changes to the IDs should be in a different PR, the sprite recolor to a different PR, the dufflebags change to a different PR (total powercreep, I honestly don't see this as being necessary at all).

In general, I really don't like the thought of "Oh yeah I'm pre-emptively buffing this for something that's still in the making." That's not how we roll.

**Comment by dawsonkeyes** [comment]
Very nice. Let’s see Paul Allen’s armory.

**Comment by GoldenAlpharex** [comment]
I don't really know what's going on in this PR anymore, please re-open PRs with the content atomized, there was good fixes, but I don't want to merge fixes bundled with things that I don't necessarily feel comfortable to have in the codenase yet. This has been dragging on for forever.
